cypress/utils/random.js

```diff
@@ -1,4 +1,38 @@
-import { visTypeDisplayNames } from '@dhis2/analytics'
+import {
+    VIS_TYPE_PIVOT_TABLE,
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_STACKED_COLUMN,
+    VIS_TYPE_BAR,
+    VIS_TYPE_STACKED_BAR,
+    VIS_TYPE_LINE,
+    VIS_TYPE_AREA,
+    VIS_TYPE_STACKED_AREA,
+    VIS_TYPE_PIE,
+    VIS_TYPE_RADAR,
+    VIS_TYPE_GAUGE,
+    VIS_TYPE_YEAR_OVER_YEAR_LINE,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_SINGLE_VALUE,
+    VIS_TYPE_SCATTER,
+} from '@dhis2/analytics'
+
+const visTypes = [
```

Comment:
I think this should be placed somewhere else than `random.js`, as it needs to be used in other places as well (which are unrelated to the generation of random items). e.g. `open` and `new` uses the old list as the source for all vis types to be tested, those will need the `visTypes` list as well.

Reply:
Moved the list to `utils/data.js` where the vis types from analytics are already imported.
Use this exported list also in `new.cy.js`.